qiita_pet/handlers/qiita_redbiom.py

```diff
@@ -71,7 +72,9 @@ def _redbiom_taxon_search(self, query, contexts):
             # find the features with those taxonomies and then search
             # those features in the samples
             features = redbiom.fetch.taxon_descendents(ctx, query)
-            for idx in redbiom.util.ids_from(features, True, 'feature', ctx):
+            if len(features) > 600:
```

Comment:
Please add comment description for this magic number.

Reply:
done